### ISSUE TITLE
.Net Developer for Avaya Genesis or Voxtron

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#7323]
.Net Developer for Avaya, Genesis, or Voxtron<br>
https://ift.tt/2I5sjIa<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0001/8469/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> Berlin and Stuttgart, Germany<br>
<strong>URL:</strong> <a href="https://virtualq.io/en/career/">https://virtualq.io/en/career/</a></p>
<div>- Remote or in one of our German offices in Berlin and Stuttgart -</div>
<div><br></div>
<div>Looking for a&nbsp;<strong>#dreamjob</strong>, where you can&nbsp;<strong>apply your .Net knowledge</strong>&nbsp;and&nbsp;<strong>your experience with Call Centers</strong>? At virtualQ you will get the chance to work on an innovative and awarded product with&nbsp;<strong>top-notch technology and software management tools</strong>. We have a German-American background and like the concept of&nbsp;<strong>simplicity, design and flat hierarchy</strong>. You have the chance to work in an awesome team atmosphere with lots for individual liberties. Do you want to change the way in which people wait on hold in customer service telephone lines with us? Then you should take your chance and join us as a .Net&nbsp;Developer!</div>
<div><br></div>
<div>
<strong>IN A NUTSHELL &ndash; WHO WE ARE AND WHAT WE DO</strong><br>
</div>
<div><br></div>
<div>virtualQ provides a&nbsp;<strong>novel cloud-solution</strong>&nbsp;that waits on hold during longer waiting times for callers of Customer Service Centers. Callers can spend their time productively until virtualQ alerts them to call back when it is their turn. Moreover, the&nbsp;<strong>smart voice assistant&nbsp;</strong>of virtualQ provides answers to frequently asked questions. Thereby, the virtualQ&nbsp;<strong>enterprise software</strong>&nbsp;creates an unique customer experience, which is currently turning the service industry upside down. On the company&acute;s side the&nbsp;<strong>smart virtualQ algorithms</strong>&nbsp;enable customer service centers to manage their incoming calls automatically and more efficiently. Well-known companies like&nbsp;<strong>Lufthansa Albatros</strong>, the German insurances&nbsp;<strong>HanseMerkur and Gothar</strong>&nbsp;and the Austrian railway provider&nbsp;<strong>&Ouml;sterreichische Bundesbahn</strong>&nbsp;are already working with our software successfully. Several national and international institutions awarded the innovative cloud-products of virtualQ and the company is backed by an institutional investment fund.</div>
<div><br></div>
<div><strong>WITH THESE ACTIVITIES YOU WILL HAVE A GREAT IMPACT ON OUR BUSINESS</strong></div>
<div><br></div>
<ul>
<li>Develop software for large Call Center and ACD systems&nbsp;</li>
<li>Work closely with large Enterprise Companies&nbsp;</li>
<li>Develop our SDK to access our REST API&nbsp;<a href="https://api.virtualq.io/api/docs#/standard">https://api.virtualq.io/api/docs#/standard</a>
</li>
<li>Connect the C# SDK of virtualQ&reg; to a Genesis, Voxtron and Avaya Call Centers</li>
<li>Work closely with our CTO and shape our product</li>
<li>Bring in your own ideas, tools, and creativity!</li>
</ul>
<div><strong><br></strong></div>
<div><strong>WITH THESE SKILLS YOU WILL BE OUR PERFECT CANDIDATE</strong></div>
<div><strong>MUST HAVES</strong></div>
<div><br></div>
<ul>
<li>Experience with Avaya, Genesis or Voxtron</li>
<li>C#/.NET / SDK Development / REST API &ndash;&nbsp;<strong>you know how to code</strong>
</li>
<li>College degree OR at least 4 years of work experience</li>
<li>You know how to communicate in an effective manner</li>
<li>You speak&nbsp;<strong>German OR English fluently</strong>&nbsp;(native or 2nd language)</li>
<li>You love working in an<strong>&nbsp;international environment</strong>
</li>
<li>You do not take things for granted, instead you question the status quo<br>
<br>
</li>
</ul>
<div><strong>WE HAVE GOT A LOT TO OFFER</strong></div>
<div><br></div>
<ul>
<li>A fair salary &ndash; we might be a startup, but we pay like a company</li>
<li>
<strong>Results Only Work Environment</strong>&nbsp;&ndash; we do not care when and where you work, we care about your work results. Therefore, we do not count working hours or holidays. Home Office Days are part of our company culture</li>
<li>Plenty of freedom and&nbsp;<strong>opportunities to grow</strong>&nbsp;in a young company, which is currently turning the Service Center market upside down</li>
<li>Workshops? Seminars? BarCamps? Tell us what you need to develop your skills and we will make it happen</li>
<li>An awesome,&nbsp;<span class="Apple-style-span">international, open minded team<br></span>
</li>
</ul>
<div><span class="Apple-style-span"><br></span></div>
<div><span class="Apple-style-span">We are an equal opportunities employer and welcome applications from all suitably qualified persons regardless of their race, sex, disability, religion/belief, sexual orientation or age.</span></div>
<div><br></div>
<div><span><b>YOUR DREAMJOB MIGHT BE JUST A CLICK AWAY. SO APPLY NOW!</b></span></div>
<div><br></div>
<div>
<b>Full job description:</b><span>&nbsp;http://bit.ly/Job_NetDeveloper</span>
</div>
<div><span>Find out more about virtualQ as an employer:&nbsp;https://ift.tt/2K52V5R</span></div>
<div><br></div>
<div>Your contact person:&nbsp;<strong>Anika</strong>
</div>
<div><br></div>
<p><strong>To apply:</strong> Please use our online application form: <a href="http://bit.ly/ApplyNow_NetDeveloper">http://bit.ly/ApplyNow_NetDeveloper</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
May 09, 2018 at 09:15AM
